runtime/compiler/runtime/RelocationRuntimeLogger.cpp

```diff
@@ -240,10 +209,16 @@ TR_RelocationRuntimeLogger::relocationTime()
 
       bool wasLocked = lockLog();
       method(false);
-      JITRT_PRINTF(jitConfig())(jitConfig(), " <%p-%p> ",
-                                             reloRuntime()->exceptionTable()->startPC,
-                                             reloRuntime()->exceptionTable()->endPC);
-      JITRT_PRINTF(jitConfig())(jitConfig(), " Time: %d usec\n", reloEndTime-_reloStartTime);
+
+      rtlogPrintf(jitConfig(), reloRuntime()->fej9()->_compInfoPT,
+                  " <%p-%p> ",
+                  reloRuntime()->exceptionTable()->startPC,
+                  reloRuntime()->exceptionTable()->endPC);
+
+      rtlogPrintf(jitConfig(), reloRuntime()->fej9()->_compInfoPT,
+                  " Time: %d usec\n",
+                  reloEndTime-_reloStartTime);
```

Comment:
This looks to be a preexisting problem, but `reloEndTime` and `_reloStartTime` are both `UDATA`, so their difference will also be `UDATA`, which on 64-bit is too big for `%d`. For `JITRT_PRINTF()`, the appropriate thing would have been `%zu`, but for `rtlogPrintf()` that won't be portable. You'll need to either cast to `int` (or better, `unsigned` with `%u`) or use `OMR_PRIuPTR`. Casting to `unsigned` should be pretty safe because at microsecond precision, it will be able to represent durations of over an hour

Reply:
Addressed in https://github.com/eclipse-openj9/openj9/pull/14210/commits/de354ea9c291db3231788240d621ed75f430322c